composer.json

```diff
@@ -18,7 +18,8 @@
         "ext-zip": "*",
         "guzzlehttp/guzzle": "~4.0|~5.0|~6.0",
         "symfony/console": "~2.3|~3.0",
-        "symfony/process": "~2.3|~3.0"
+        "symfony/process": "~2.3|~3.0",
+        "symfony/filesystem": "^3.3"
```

Comment:
Also, please put this in between console and process, so it's in the correct order.